.circleci/config.yml

```diff
@@ -503,7 +503,7 @@ jobs:
     executor: bionic
     steps:
       - run-tests:
-          test_targets: "wasm2"
+          test_targets: "wasm2 asan.test_embind*"
```

Comment:
Do we want to rename this job ?  Or create a new asan job?

Reply:
It's just a handful of tests, so a new job seems excessive.

Renaming seems possible, but I don't have a great idea...?

Reply:
Can't think of a good one :(